cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -397,58 +384,45 @@ components:
             enum:
               - "cell"
               - "nucleus"
-          - # If assay_ontology_term_id is EFO:0009918 or its descendants, 'suspension_type' MUST be 'na'
+          - # If assay_ontology_term_id is EFO:0008994 or its descendants, 'suspension_type' MUST be 'na'
```

Comment:
should this be [EFO:0008992](https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008992)?